src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/ViewResultExecutor.cs

```diff
@@ -68,7 +68,12 @@ public virtual ViewEngineResult FindView(ActionContext actionContext, ViewResult
             var viewEngine = viewResult.ViewEngine ?? ViewEngine;
             var viewName = viewResult.ViewName ?? actionContext.ActionDescriptor.Name;
 
-            var result = viewEngine.FindView(actionContext, viewName);
+            var result = viewEngine.GetView(executingFilePath: null, viewPath: viewName, isPartial: false);
```

Comment:
Isn't this essentially moving the check one layer up (out of the `IViewEngine`)? For any `IViewEngine` lookup aren't you essentially forced to perform both these checks (or is there a scenario where you can only have to call `GetView` \ `FindView` but not the other?